.github/workflows/build_navitia_packages_for_release.yml

```diff
@@ -35,3 +35,11 @@ jobs:
       run: rm -rf ../navitia-*
     - name: trigger jenkins to publish Artifacts on internal server
       run: http -v -f POST https://${{secrets.jenkins_token}}@ci.navitia.io/job/publish_navitia_release_packages_from_github/build
+    - name: slack notification (the job has failed)
+      if: failure()
+      run: |
+          echo '{"text":":warning: Github Actions: build_navitia_packages_for_release failed ! (https://github.com/CanalTP/navitia/actions?query=workflow%3A%22Build+Navitia+Packages+For+Release%22). Navboys, this is a release alert !!!"}' | http --json POST ${{secrets.SLACK_URL}}
+    - name: slack notification (the job has successed)
+      if: success()
+      run: |
+          echo '{"text":":tada: Github Actions: build_navitia_packages_for_release succeded - navitia debian packages are available"}' | http --json POST ${{secrets.SLACK_URL}}
```

Comment:
Just for Bonus point, do you think we can have the version number being released ? Would be nice to have that in the #Navitia chanel as well.

Reply:
Yes I can add a message ! 
I'll see if it is possible to add a **automatic message** wiith **version number** and a link to the the **changelog**.